suites/configurations/inputs-aws-ec2.yaml

```diff
@@ -0,0 +1,18 @@
+aws_access_key_id: ''
+aws_secret_access_key: ''
+image_id: ''
+instance_type: ''
+manager_keypair_name: 'shared-systemt-tests-key'
+agent_keypair_name: 'shared-systemt-tests-key'
+
+use_existing_manager_group: false
+use_existing_agent_group: false
+use_existing_manager_keypair: false
```

Comment:
`use_existing_manager_keypair` and `use_existing_agent_keypair` should be `true`